### PR TITLE
benchmark: add txn-put benchmark

### PR DESCRIPTION
**Pull request by heyitsanthony**
Submits multiple put ops in a single txn.

### CODE REVIEW

File: tools/benchmark/cmd/txn_put.go [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+				bar.Increment()
+			}
+		}(clients[i])
+	}
+
+	go func() {
+		for i := 0; i < txnPutTotal; i++ {
+			ops := make([]v3.Op, txnPutOpsPerTxn)
+			for j := 0; j < txnPutOpsPerTxn; j++ {
+				binary.PutVarint(k, int64(i%keySpaceSize))
```
Comment (gyuho) [inline tools/benchmark/cmd/txn_put.go]: Did you mean `j%keySpaceSize`? If not, define before the loop?

Comment (heyitsanthony) [inline tools/benchmark/cmd/txn_put.go]: Oh, it should be `((i*txnPutOpsPerTxn)+j) % keySpaceSize` since I want unique keys. Thanks!

**Comment by xiang90** [comment]
any example result?

**Comment by heyitsanthony** [comment]
hd run, 1 node, fresh member dir each run:
```
$ ./benchmark --conns=128 --clients=128 txn-put --txn-ops=4 --key-space-size=1000000 --total=10000
 10000 / 10000 Boooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo! 100.00%3s

Summary:
  Total:	3.7849 secs.
  Slowest:	0.1854 secs.
  Fastest:	0.0015 secs.
  Average:	0.0477 secs.
  Stddev:	0.0231 secs.
  Requests/sec:	2642.0436

Response time histogram:
  0.0015 [1]	|
  0.0199 [221]	|â
  0.0383 [1362]	|âââââââ
  0.0567 [7520]	|ââââââââââââââââââââââââââââââââââââââââ
  0.0751 [35]	|
  0.0935 [342]	|â
  0.1119 [242]	|â
  0.1303 [100]	|
  0.1487 [49]	|
  0.1670 [0]	|
  0.1854 [128]	|

Latency distribution:
  10% in 0.0350 secs.
  25% in 0.0405 secs.
  50% in 0.0422 secs.
  75% in 0.0483 secs.
  90% in 0.0526 secs.
  95% in 0.0951 secs.
  99% in 0.1838 secs.
  99.9% in 0.1849 secs.
```

```

$ ./benchmark --conns=128 --clients=128 txn-put --txn-ops=64 --key-space-size=1000000 --total=10000
 10000 / 10000 Booooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo! 100.00%15s

Summary:
  Total:	15.5049 secs.
  Slowest:	0.6095 secs.
  Fastest:	0.0044 secs.
  Average:	0.1967 secs.
  Stddev:	0.0901 secs.
  Requests/sec:	644.9562

Response time histogram:
  0.0044 [1]	|
  0.0649 [1124]	|âââââââââ
  0.1254 [603]	|ââââ
  0.1859 [1803]	|ââââââââââââââ
  0.2464 [4973]	|ââââââââââââââââââââââââââââââââââââââââ
  0.3069 [1091]	|ââââââââ
  0.3674 [143]	|â
  0.4280 [6]	|
  0.4885 [0]	|
  0.5490 [128]	|â
  0.6095 [128]	|â

Latency distribution:
  10% in 0.0630 secs.
  25% in 0.1512 secs.
  50% in 0.2106 secs.
  75% in 0.2297 secs.
  90% in 0.2612 secs.
  95% in 0.2960 secs.
  99% in 0.6055 secs.
  99.9% in 0.6092 secs.
```

```
$ ./benchmark --conns=128 --clients=128 txn-put --txn-ops=128 --key-space-size=1000000 --total=10000
 10000 / 10000 Booooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo! 100.00%29s

Summary:
  Total:	29.6294 secs.
  Slowest:	0.9075 secs.
  Fastest:	0.0216 secs.
  Average:	0.3778 secs.
  Stddev:	0.1123 secs.
  Requests/sec:	337.5031

Response time histogram:
  0.0216 [1]	|
  0.1101 [76]	|
  0.1987 [70]	|
  0.2873 [1940]	|ââââââââââââââââââââââ
  0.3759 [2723]	|âââââââââââââââââââââââââââââââ
  0.4645 [3483]	|ââââââââââââââââââââââââââââââââââââââââ
  0.5531 [1199]	|âââââââââââââ
  0.6417 [321]	|âââ
  0.7303 [59]	|
  0.8189 [67]	|
  0.9075 [61]	|

Latency distribution:
  10% in 0.2497 secs.
  25% in 0.2936 secs.
  50% in 0.3792 secs.
  75% in 0.4454 secs.
  90% in 0.4875 secs.
  95% in 0.5548 secs.
  99% in 0.8174 secs.
  99.9% in 0.9053 secs.
```

**Comment by xiang90** [comment]
lgtm

**Pull request merged**